docs/pages/reference/workload-identity/workload-identity-api-and-workload-attestation.mdx

```diff
@@ -282,6 +306,143 @@ spec:
+The Podman Workload Attestor allows you to restrict the issuance of workload
+identities to specific Podman containers and pods based on a range of criteria.
+
+It works by first determining the container and pod for a given process and then
+by querying the [Podman API Service](https://docs.podman.io/en/latest/markdown/podman-system-service.1.html)
+for details about them.
+
+Podman differs from Docker in that it doesn't use a long-running daemon process.
+Instead, the API service is typically started on-demand using socket activation.
+
+Granting `tbot` access to the Podman API requires different steps depending on
+whether you're running "rootfully" or "rootlessly" - it is for this reason we
+do not provide a default value for the `attr` config option.
+
+### Rootful Podman
+
+If you're running Podman as root, enable the API service by running:
+
+```shell
+sudo systemctl enable --now podman.socket
+```
+
+At this point, the service can be reached at `/run/podman/podman.sock` but **only
+by root**. If you do not want to run `tbot` as root, you can override the default
+systemd unit to make the socket owned by a group your tbot user belongs to:
+
+```shell
+sudo groupadd podman-root
+sudo usermod -a -G podman-root <tbot user>
+sudo systemctl edit podman.socket
+```
+
+In the override file add:
+
+```
+[Socket]
+SocketGroup=podman-root
+```
+
+Reload the systemd daemon and restart the socket unit:
+
+```shell
+sudo systemctl daemon-reload
+sudo systemctl restart podman.socket
+```
+
+### Rootless Podman
+
+If you're running Podman as a non-root user, enable and start the API service by
+running:
+
+```shell
+sudo loginctl enable-linger <podman user>
+systemctl --user enable --now podman.socket
+```
+
+The service can now be reached at `$XDG_RUNTIME_DIR/podman/podman.sock`, but
+**only by your chosen podman user or root**. If you do not want to run `tbot`
+as the same user or as root, you can override the default systemd unit to move
+the socket out of `$XDG_RUNTIME_DIR` and add your tbot user to the podman user's
+group:
+
+```shell
+sudo usermod -a -G <podman user group> <tbot user>
+systemctl --user edit podman.socket
+```
+
+In the override file add:
+
+```
+[Socket]
+ListenStream=/path/to/socket # Example: /srv/podman.<podman user>/podman.sock
+```
+
+Reload the systemd daemon and restart the socket unit:
+
+```shell
+systemctl --user daemon-reload
+systemctl --user restart podman.socket
+```
+
+### Running `tbot` in a Podman Container
+
+In order for `tbot` running inside a container to attest workloads in other
+containers, it must have access to the host's PID namespace. Also, if you are
+using group permissions to grant `tbot` access to the Podman socket (see the
+above examples) you must set the `run.oci.keep_original_groups` annotation.
+
+```shell
+podman run \
+  --pid=host \
+  --annotation run.oci.keep_original_groups=1 \
+  --volume /path/to/socket:/path/to/socket \
+  --volume /path/to/config:/path/to/config \
+  <tbot image> \
```

Comment:
Is there a way we could populate the `<tbot image>` placeholder within the text to make copying and pasting easier? For example, we could use the `Var` component, e.g.,:

````markdown
Assign <Var name="tbot-image" /> to the name of the `tbot` Docker image:

```code
podman run \
...
   <Var name="tbot-image" />
```
````

Reply:
Good question! Having searched around a little, I think we can replace it with this:

```
public.ecr.aws/gravitational/tbot-distroless:(=teleport.version=)
```